Remora.Plugins.Abstractions/Remora.Plugins.Abstractions.csproj

```diff
@@ -1,17 +1,17 @@
-<Project Sdk="Remora.Sdk/1.0.0-preview6">
-    <PropertyGroup>
-        <Version>5.0.0</Version>
-        <Description>
-            Abstract representations of a plugin system.
-        </Description>
-        <PackageReleaseNotes>
-            Upgrade Remora.Results.
-        </PackageReleaseNotes>
-        <PackageTags>$(PackageTags);plugin;plugins;dynamic</PackageTags>
-    </PropertyGroup>
-
-    <ItemGroup>
-      <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
-      <PackageReference Include="Remora.Results" Version="7.2.1" />
-    </ItemGroup>
-</Project>
+﻿<Project Sdk="Remora.Sdk/1.0.0-preview6">
+    <PropertyGroup>
+        <Version>5.0.0</Version>
+        <Description>
+            Abstract representations of a plugin system.
+        </Description>
+        <PackageReleaseNotes>
+            Upgrade Remora.Results.
+        </PackageReleaseNotes>
+        <PackageTags>$(PackageTags);plugin;plugins;dynamic</PackageTags>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />

```

Comment:
.BuildServiceProvider() seems to not be in ``.Abstractions`` package so this had to be changed as well.